source/agora/node/main.d

```diff
@@ -63,25 +63,30 @@ private int main (string[] args)
             ex.message);
     }
 
-    NodeListenerTuple node_listener_tuple;
-    try
-    {
-        auto config = parseConfigFile(cmdln);
-        if (cmdln.config_check)
+    bool configError;
```

Comment:
Not according to our style guide..

Reply:
![Oops](https://media2.giphy.com/media/vX9WcCiWwUF7G/giphy.gif)